### PR TITLE
D8 nid 1363

### PR DESCRIPTION
**Pull request by DuttonMa**
Install dblog module

### CODE REVIEW

File: config/sync/ultimate_cron.job.update_cron.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+uuid: bea581c7-b334-423c-b122-aa68c873c937
+langcode: en
+status: true
+dependencies:
+  module:
+    - update
+title: 'Update indexes'
```
Comment (johangant) [inline config/sync/ultimate_cron.job.update_cron.yml:7]: Is this required for dblog to operate?

File: config/sync/dblog.settings.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+row_limit: 1000
```
Comment (johangant) [inline config/sync/dblog.settings.yml:1]: I'd suggest a slightly higher amount... 10k? CSP reports would fill this rather quickly in the short term. 10k rows shouldn't take up much disk space on the db service.

File: config/sync/ultimate_cron.job.update_cron.yml (hunk 1 above)
Comment (DuttonMa) [reply config/sync/ultimate_cron.job.update_cron.yml:7]: Good question, I'll try deleting it

File: config/sync/dblog.settings.yml (hunk 2 above)
Comment (DuttonMa) [reply config/sync/dblog.settings.yml:1]: Yes, another good spot !

**Review by johangant — APPROVED** [review]
LGTM

**Pull request merged**